src/services/authentication/hooks/index.js

```diff
@@ -1,5 +1,8 @@
-const { TooManyRequests } = require('@feathersjs/errors');
+const { TooManyRequests, NotAuthenticated } = require('@feathersjs/errors');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['NotAuthenticated' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4141092)